src/DETHRACE/pc-win95/win95sys.c

```diff
@@ -308,7 +308,7 @@ void Win32ReleaseInputDevice(void) {
 int PDGetASCIIFromKey(int pKey) {
     LOG_TRACE("(%d)", pKey);
 
-    if (PDKeyDown(KEY_SHIFT_ANY)) {
+    if (PDKeyDown3(KEY_SHIFT_ANY)) {
```

Comment:
lets put a comment here describing why we are deviating from the OG windows code